t.py

```diff
@@ -0,0 +1,86 @@
+#coding=utf-8
+
+from flask import Flask
+import requests
+import json
+import sys
+import time
+import getpass
+
+USERNAME = "" 
+PASSWORD = ""
+
+s = "" # session
+
+URL = "http://kuas.grd.idv.tw:14768/" #API URL
+
+def setData():
+    global USERNAME,PASSWORD
+    USERNAME = raw_input("Enter Your Account:")
+    PASSWORD = getpass.getpass("Enter Your Password:")
```

Comment:
You should remove the logic of getting user name and password out of this function, since your function name is `sendData`, it shouldn't bother the problem of getting username and password.